accounting/serializers.py

```diff
@@ -16,4 +21,46 @@ class Meta:
             'channel_name',
             'date_created',
             'date_updated'
-        )
\ No newline at end of file
+        )
+
+class ProjectDetailsSerializer(serializers.ModelSerializer):
+    """ project serializer
+    """
+    date_data = serializers.SerializerMethodField()
+    pending_issues = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Project
+        fields = (
+            'id',
+            'name',
+            'date_data',
+            'description',
+            'channel_name',
+            'date_created',
+            'date_updated',
+            'pending_issues'
+        )
+
+    def get_date_data(self, obj):
```

Comment:
is this possible to be calculated in the frontend? this serves no purpose in the backend